ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -65,7 +65,7 @@ listen_addresses = '*'      # what IP address(es) to listen on;
                     # defaults to 'localhost', '*' = all
                     # (change requires restart)
 port = 5432             # (change requires restart)
-max_connections = 100           # (change requires restart)
+max_connections = 300           # (change requires restart)
```

Comment:
wasn't super sure on how to do a secure transfer, so i just `scp`d it onto hqdb0 under /home/brudolph. it's 80MB so it'll probably take your browser a bit to load. also feel free to delete once you've copied it. i have a local copy